src/main/groovy/com/github/jengelman/gradle/plugins/shadow/relocation/SimpleRelocator.groovy

```
@@ -151,6 +151,10 @@ class SimpleRelocator implements Relocator {
 
     @Override
     boolean canRelocatePath(String path) {
+        if (path.isEmpty()) {
+            return false
+        }
```

We can probably throw an exception here in this case.